base/cvd/cuttlefish/host/commands/cvd/parser/load_configs_parser.cpp

```diff
@@ -271,6 +273,29 @@ Result<LoadDirectories> GenerateLoadDirectories(
   return result;
 }
 
+std::vector<std::string> GenerateMissingInstanceNames(
```

Comment:
WDYT of `void FillEmptyInstanceNames(std::vector<std::string>& names)`?

With `GenerateMissingInstanceNames`, at a first reading I thought we were generating only the missing names, although we return the existing ones as well. If keeping this name, let's add a brief comment explaining this to the function.

Reply:
Done